e2e/cypress/support/api_commands.js

```diff
@@ -117,6 +117,18 @@ Cypress.Commands.add('apiPatchChannel', (channelId, channelData) => {
     });
 });
 
+Cypress.Commands.add('apiPatchChannelPrivacy', (channelId, privacy = 'O') => {
```

Comment:
We're organizing Cypress commands, especially API, and are preventing from adding more to existing `support/api_commands.js`. Please create new file named `support/api/channel.js` and move it there. Also, please add type to `support/api/channel.d.ts`.

Reply:
Done! Thanks :)